components/playback/controls/PlateBarcode.vue

```diff
@@ -6,69 +6,49 @@
     <!-- original Mockflow ID: cmpDd0be63536ca605546f566539e51ad0c3-->
     <input
       id="plateinfo"
-      :disabled="
-        playback_state === playback_state_enums.RECORDING ||
-        playback_state === playback_state_enums.BUFFERING ||
-        playback_state === playback_state_enums.LIVE_VIEW_ACTIVE
-      "
+      disabled="disabled"
       type="text"
       spellcheck="false"
       onpaste="return false;"
```

Comment:
this `onpaste` is no longer necessary, correct?

Reply:
Removed